packages/components/src/form-token-field/suggestions-list.js

```diff
@@ -2,157 +2,147 @@
  * External dependencies
  */
 import { map } from 'lodash';
-import scrollIntoView from 'dom-scroll-into-view';
+import scrollView from 'dom-scroll-into-view';
 import classnames from 'classnames';
 
 /**
  * WordPress dependencies
  */
-import { Component } from '@wordpress/element';
+import { useEffect, useState } from '@wordpress/element';
 import { withSafeTimeout } from '@wordpress/compose';
 
-class SuggestionsList extends Component {
-	constructor() {
-		super( ...arguments );
-		this.handleMouseDown = this.handleMouseDown.bind( this );
-		this.bindList = this.bindList.bind( this );
-	}
+function SuggestionsList( {
+	selectedIndex,
+	scrollIntoView,
+	match = '',
+	onHover = () => {},
+	onSelect = () => {},
+	suggestions = Object.freeze( [] ),
```

Comment:
`Object.freeze( [] )` could be moved out of this function into a `const` to avoid recreating it on every render.